sync/command.py

```diff
@@ -712,7 +722,7 @@ def do_landable(git_gecko,
                 if sync.error:
                     msg = sync.error["message"] or ""
                     msg = "%s (%s)" % (msg, msg.split("\n")[0])
-            print("%s: %s" % (pr, msg))
+            print("%s: %s" % (pr, msg.splitlines()[0]))
```

Comment:
nit: any reasons to not using `splitlines()` twice here 

```
>>> lines = "line 1\nline 2\nline 3\n"
>>> lines.split('\n')[0]
'line 1'
>>> lines.splitlines()[0]
'line 1'
```

Reply:
I think what I"m doing here doesn't make any sense. Good catch :)